src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -165,7 +165,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler()
     //
     // Mark request as websocket if upgrade header is present.
     //
-    if (g_fWebSocketSupported)
+    if (pApplication->QueryConfig()->QueryWebSocketEnable())
```

Comment:
I think we should still check win8+ too?

Reply:
Remove g_fWebSocketSupported.

Reply:
no need to check win8 as on the websocket configuration section is empty win7 and QueryWebSocketEnable return false. g_fWebSocketedSupported is used for initialize winhttpwebsocket function pointers and static init/cleanup

Reply:
Then shouldn't g_fWebSocketedSupported be equal to pApplication->QueryConfig()->QueryWebSocketEnabled()?